default/content/presets/instruct/Gemma 2.json

```diff
@@ -0,0 +1,24 @@
+{
+    "system_prompt": "Continue writing this story and portray characters realistically.",
+    "input_sequence": "<start_of_turn>user",
+    "output_sequence": "<start_of_turn>model",
+    "last_output_sequence": "",
+    "system_sequence": "<start_of_turn>system",
```

Comment:
Note: system message wrapping is disabled and routes to user since Gemma has no concept of system messages at all. Brave enough may try to disable that guardrail and see if it behaves better in response to system instructions in this case.